lib/discovery.js

```diff
@@ -427,7 +429,7 @@ export function storeAccountDiscovery(
     let data = {
         sources: sources.map((source) => source.store()),
         histories: account.map((state) => state.history),
-        transactionIds: txs.ids(),
+        transactionIds: Array.from(txs.keys()),
```

Comment:
this is only here because I have split my changes into two PRs; this line is gone in the other PR since I put transactionIds out